otx/algorithms/detection/adapters/mmdet/utils/builder.py

```diff
@@ -28,8 +28,6 @@ def build_detector(
 
     from mmdet.models import build_detector as origin_build_detector
 
-    model_cfg = deepcopy(config.model)
```

Comment:
Could you explain why this part is removed?

Reply:
I will revert this change to avoid conflict as Inhyuk also found this issue in his PR

https://github.com/openvinotoolkit/training_extensions/pull/1553/files#diff-d9bdf7888477389f59524b4f06c95e46d030d1fd25191158dc6a9b6dd1188f6bL31